src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -146,3 +146,49 @@ def irdft(
         inputs = as_nodes(data, axes, signal_size)
 
     return _get_node_factory_opset9().create("IRDFT", inputs)
+
+
+@nameable_op
+def generate_proposals(
+    im_info: NodeInput,
+    anchors: NodeInput,
+    deltas: NodeInput,
+    scores: NodeInput,
+    min_size: float,
+    nms_threshold: float,
+    post_nms_count: int,
+    pre_nms_count: int,
+    normalized: bool = True,
+    nms_eta: float = 1.0,
+    roi_num_type: str = "i64",
+    name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs GenerateProposals operation.
+
+    :param im_info: Tensor with input image info.
+    :param anchors: Tensor with input anchors.
+    :param deltas: Tensor with input deltas.
+    :param scores: Tensor with input scores.
+    :param min_size: Specifies minimum box width and height.
+    :param nms_threshold: Specifies threshold to be used in the NMS stage.
+    :param pre_nms_count: Specifies number of top-n proposals before NMS.
+    :param post_nms_count: Specifies number of top-n proposals after NMS.
+    :param normalized: Specifies whether proposal bboxes are normalized or not.
+    :param nms_eta: Specifies eta parameter for adaptive NMS.
+    :param roi_num_type: Specifies element type of output roi_num.
+    :param name: The optional name for the output node.
+    :return: New node performing GenerateProposals operation.
+    """
+    inputs = as_nodes(im_info, anchors, deltas, scores)
+
+    attributes = {
+        "min_size": min_size,
+        "nms_threshold": nms_threshold,
+        "post_nms_count": post_nms_count,
+        "pre_nms_count": pre_nms_count,
+        "normalized": normalized,
+        "nms_eta": nms_eta,
+        "roi_num_type": roi_num_type
```

Comment:
Is it correct to pack it into attributes? C++ implementation is giving it a specific slot in constructor. Please check it, if it somehow packs into attributes anyway, I am OK with it.

Reply:
Yes, it works. And it is checked in https://github.com/meiyang-intel/openvino/blob/c74ec58b2c947a284106141b7bb05293ac58e34c/src/bindings/python/tests/test_ngraph/test_create_op.py#L2138.